node/actors/bft/src/leader/mod.rs

```diff
@@ -8,4 +8,8 @@ mod state_machine;
 #[cfg(test)]
 mod tests;
 
+#[allow(unused_imports)]
+pub(crate) use replica_commit::Error as ReplicaCommitError;
```

Comment:
Off-topic (?): IMO, it would make sense to prefix such re-exports with `self::` to clearly distinguish them from imports / re-exports from other crates.

Reply:
Fixed: https://github.com/matter-labs/era-consensus/pull/39/commits/1aee2fecbac8a38f925d49077551a360fd27fa20